frontend/package.json

```diff
@@ -11,5 +11,8 @@
   "license": "ISC",
   "devDependencies": {
```

Comment:
Sorry for the drive-by comment here, but it would be great to add eslint, prettier, and any other tools we can to validate the front end code similar to how we're doing for Python. Happy to talk through setting that up!

Reply:
Just added eslint and prettier!

Reply:
Awesome! Do you want to add those to the GitHub Actions check as well? You could most likely copy this as-is in a separate job so that it runs in parallel with the Python checks https://github.com/CMSgov/hpt-tool/blob/d9ca817da78667610f45263cdabc8f3b0a57a956/.github/workflows/ci.yml#L21-L45

Reply:
True, just added! Didn't realize it was that easy to include. This is also missing in the metrics website so I'll add it there too.

Reply:
Awesome, looks great!